src/Console/Command/DebugUnassignedCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Console\Command;
+
+use Qossmic\Deptrac\Configuration\Loader;
+use Qossmic\Deptrac\Console\Command\Exception\SingleDepfileIsRequiredException;
+use Qossmic\Deptrac\Console\Symfony\Style;
+use Qossmic\Deptrac\UnassignedAnalyser;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DebugUnassignedCommand extends Command
+{
+    protected static $defaultName = 'debug:unassigned';
```

Comment:
Symfony 5.3 introduced lazy command descriptions. See: https://symfony.com/blog/new-in-symfony-5-3-lazy-command-description

Maybe it makes sense to add one as well? Using the text from the README would be good, e.g.

```
protected static $defaultDescription = 'Lists all classes from your paths that are not assigned to any layer.';